src/App.js

```diff
@@ -106,11 +107,23 @@ export default class App extends Component {
    */
   renderTooltip() {
     const {hoveredCounty, hoveredZipCode, x, y} = this.state;
+    const style = {
```

Comment:
Is there a reason you don't want to give the components classNames and put the styles in a CSS file? Just thinking about cleaning it up a little.

Reply:
There actually are two named components that replace that: ZipCodeSnapshotOverlay and CountySnapshotOverlay. They are implemented by Map.js, which will take the mapping logic out of App.js. I had left App.js pretty much alone for the time being, until we had sorted out the state-management situation. I think a following ticket will be to refactor App.js

Reply:
Ah okay, thanks!